README.rst

```diff
@@ -1,64 +1,44 @@
-===============
-Getting started
-===============
-
-------------------------------------------------
-Install dependencies & start the service locally
-------------------------------------------------
-
-1. Enter the project folder:
-
-.. code-block:: console
-
-    $ cd flexprep
-
-2. Install packages
-
-.. code-block:: console
-
-    $ poetry install
-
-
-3. Run the job
-
-.. code-block:: console
-
-    $ poetry run python -m flexprep
-
--------------------------------
-Run the tests and quality tools
--------------------------------
-
-1. Run tests
-
-.. code-block:: console
-
-    $ poetry run pytest
-
-2. Run pylint
-
-.. code-block:: console
-
-    $ poetry run pylint flexprep
-
-
-3. Run mypy
-
-.. code-block:: console
-
-    $ poetry run mypy flexprep
-
-
-----------------------
-Generate documentation
-----------------------
-
-.. code-block:: console
-
-    $ poetry run sphinx-build doc doc/_build
-
-Then open the index.html file generated in *flexprep/build/sphinx/html*
-
-
-.. HINT::
-   All **poetry run** prefixes in the commands can be avoided if running them within the poetry shell
+# FlexPrep
+### Container which pre-processes IFS data for [Flexpart](https://www.flexpart.eu/) in the European Weather Cloud ([EWC](https://europeanweather.cloud/)).
+
+FlexPrep is part of an automated system triggered by product dissemination via [ECPDS](https://confluence.ecmwf.int/pages/viewpage.action?pageId=228871373) using [Aviso](https://confluence.ecmwf.int/display/EWCLOUDKB/Aviso+Notification+System+on+EWC). This setup ensures that data processing is initiated automatically when new data is available. For detailed instructions on configuring and setting up Aviso for this system, please refer to the [aviso folder in this repository.](https://github.com/MeteoSwiss-APN/flexprep/blob/main/aviso/README.md).
+
+
+## Setup Instructions for VM on EWC
+
+To set up the VM for this project, you need to manually add the following files and folders:
+
+1. **VM Data Flavor:**
+    - The VM must be provisioned with the appropriate `data` flavor ([How to provision a new instance on EWC](https://confluence.ecmwf.int/display/EWCLOUDKB/Provision+a+new+instance+-+web)). This ensures that the Aviso package is automatically installed.
```

Comment:
What is the appropriate data flavor?

Reply:
It's the layout one can select when provisioning an instance with the EWC console. For example for our VM, I chose: ubuntu-22.04-data. An example of how it can look like here on step N°4 of these [instructions](https://confluence.ecmwf.int/display/EWCLOUDKB/Provision+a+new+instance+-+web ). 
I changed the text so that I don't use the "flavour" word which is not adequate and added this text:

> When provisioning the VM on EWC following these [instructions](https://confluence.ecmwf.int/display/EWCLOUDKB/Provision+a+new+instance+-+web), ensure you select the instance with the `-data` layout (e.g. ubuntu-22.04-data).

Is it better explained?

Reply:
Much better, thanks